predictor_robojudge.ipynb

```diff
@@ -328,10 +279,6 @@
     "\n",
     "countries = None\n",
     "scenario_df = generate_scenario(start_date_str, end_date_str, latest_df, countries, scenario=\"Freeze\")\n",
-    "# Remove countries that weren't there at the beginning of the challenge or duplicates\n",
```

Comment:
The list of countries and regions is now specified in `load_dataset` at the beginning of the notebook.